cycler.py

```diff
@@ -335,7 +335,11 @@ def __repr__(self):
     def _repr_html_(self):
         # an table showing the value of each key through a full cycle
         output = "<table>"
-        sorted_keys = sorted(self.keys)
+        try:
+            sorted_keys = sorted(self.keys)
```

Comment:
can't do that because the string representation aren't keys themselves. You would have to pass the string reprs as a key to sorted.